File: JS/QuadradoAoCubo.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */

let N = parseInt( gets() );
let ar = loadSquareAndCumbe( N );

ar.forEach(function(ops) {
    console.log( `${ops[0]} ${ops[1]} ${ops[2]}` );
});

/**
 * Load all square and cumbe numbers.
 */
function loadSquareAndCumbe(n = 0) {
    let results = new Array();

    for ( let i=1; i <= n; i++ ) {
        results.push( [i, Math.pow(i, 2), Math.pow(i, 3)] );
    }

    return results;
}
module.exports = {loadSquareAndCumbe}

File: GO/QuadradoAoCubo.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package GO

import "math"

/**
 * Load all square and cumbe numbers.
 */
func loadSquareAndCumbe(n int) [][]int {
	results := [][]int{}

	for i := 1; i <= n; i++ {
		results = append(results, []int{i, int(math.Pow(float64(i), 2)), int(math.Pow(float64(i), 3))})
	}

	return results
}
